qemu/tests/qemu_guest_agent.py

```diff
@@ -195,13 +195,7 @@ def gagent_create(self, params, vm, *args):
         gagent_serial_type = args[0]
         gagent_name = args[1]
 
-        if gagent_serial_type == guest_agent.QemuAgent.SERIAL_TYPE_VIRTIO:
-            filename = vm.get_virtio_port_filename(gagent_name)
-        elif gagent_serial_type == guest_agent.QemuAgent.SERIAL_TYPE_ISA:
-            filename = vm.get_serial_console_filename(gagent_name)
-        else:
-            raise guest_agent.VAgentNotSupportedError("Not supported serial"
-                                                      " type")
+        filename = vm.get_serial_console_filename(gagent_name)
```

Comment:
@yanan-fu Please help review the update, thanks.

Reply:
LGTM

Test is running now, will update result later.

Reply:
Run gating test loop,  all pass.